client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
+		isPrivateSite: ( site ) => isPrivateSite( state, site.ID ),
+		isJetpackSite: ( site ) => isJetpackSite( state, site.ID ),
```

Comment:
Also, does it make sense for these to accept a `siteId` instead of the entire `site` object, since only the ID will be used?